### ISSUE TITLE
The img folder refers to the folder that contains the images (in jpg format). If that folder is not named after "img", than changing the folder name into "img" should solve the problem.

### ISSUE THREAD

**Issue by Sasukiii** [ipl-uw/AIC23_Track1_UWIPL_ETRI#18]
The img folder refers to the folder that contains the images (in jpg format). If that folder is not named after "img", than changing the folder name into "img" should solve the problem.

_Originally posted by @hsiangwei0903 in https://github.com/ipl-uw/AIC23_Track1_UWIPL_ETRI/issues/15#issuecomment-1823601234_

**Comment by Sasukiii** [comment]
Then I know what went wrong. When you extract frame, you only extract the frames of the training set but don't extract the video frames of the test set. Is there a missing file or a missing step when you upload the project?

**Comment by yangchris11** [comment]
You are correct; we did not include the extraction part of the testing; it can be easily modified from the codes in `scripts/preprocessing.`
